subsys/bluetooth/audio/broadcast_source.c

```diff
@@ -769,7 +769,15 @@ int bt_audio_broadcast_source_reconfig(struct bt_audio_broadcast_source *source,
 
 	SYS_SLIST_FOR_EACH_CONTAINER(&source->subgroups, subgroup, _node) {
 		SYS_SLIST_FOR_EACH_CONTAINER(&subgroup->streams, stream, _node) {
+			struct bt_iso_chan_io_qos *iso_qos;
```

Comment:
Declare closer to reference?

```suggestion
	SYS_SLIST_FOR_EACH_CONTAINER(&source->subgroups, subgroup, _node) {
		struct bt_audio_stream *stream;
		
		SYS_SLIST_FOR_EACH_CONTAINER(&subgroup->streams, stream, _node) {
			struct bt_iso_chan_io_qos *iso_qos;
```

Reply:
Would make sense, but would rather not do any (unrelated and) unnecessary changes :)